course/frontend/has-symbol/scripts/has-symbol.js

```diff
@@ -0,0 +1,10 @@
+//Function that detects if a string has any symbol (e.g. hasSymbol('hello%') => true).
+// RETURN true if any, otherwise false.
+'use strict'
+function hasSymbol( _string ){
+
+	if ( typeof _string !== 'string' ||	 _string.length === 0 || (/[\n\t]/g).test( _string) ) return 'ERROR: OUT of SPECS' ;
```

Comment:
Hello @EdgarAyllon, to control and create new errors, it is better to use `try catch`. In standard programming, the definition of variables is made immediately after declaring a function

Reply:
Yes, I know, thanks. Note as been taked for the next challenge.